lib/data_update_scripts/20201223013903_set_proper_display_area_for_profile_fields.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class SetProperDisplayAreaForProfileFields
```

Comment:
Hm, I thought we had already done something like this in the past and also added display areas to `lib/data/dev_profile_fields.csv`, so if you really want to change it, please change it there too, they should be in sync, at lest for now.

Reply:
Sounds good. I haven't investigated fully why these weren't set properly but my guess would maybe be enum trying to be set as string somewhere and failed silently?

Reply:
Strings work too:

```ruby
ProfileField.update_all(display_area: 'header')
# ProfileField Update All (4.4ms)  UPDATE "profile_fields" SET "display_area" = $1  [["display_area", 0]]
****